core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1146,7 +1146,10 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
   ): BucketMetricsResponse = {
     val metrics = ltspResponse.iterateAll().asScala.toSeq.flatMap { timeSeries =>
       timeSeries.getPointsList.asScala.map { point =>
-        BucketMetric(timeSeries.getMetric.getLabelsMap.get("storage_class"), Math.round(point.getValue.getDoubleValue))
+        BucketMetric(timeSeries.getMetric.getLabelsMap.get("storage_class"),
+                     timeSeries.getMetric.getLabelsMap.get("type"),
+                     Math.round(point.getValue.getDoubleValue)
+        )
```

Comment:
The soft-deleted-object vs live-object value was already being returned from the query; we were just ignoring it.